verify.sh

```diff
@@ -37,7 +37,8 @@ function verify_bcc() {
 
 function verify_cluster() {
     # basic check for k8s cluster info
-    if [ $(kubectl cluster-info) !=0 ]; then
+    kubectl cluster-info
+    if [ "$?" != 0 ]; then
```

Comment:
Consider these 2 alternatives 

1. My preference - use `&& ||` 
```
kubectl cluster-info || { echo "failed .. "; return 1; }
```
in #31 I have a generic `die` method that exits 1

2. No need to use `if [` 

```
if ! kubectl cluster-info ; then 
  ... 
fi
```

Reply:
Cool, I like the first one :-) Let me change as you suggested then. Thanks.
The initial reason for me to change here is just removing the original console warning.

Reply:
Alternatively, since #31 is approved, it would be great if you could rebase the major changes on top of #31.